website/docs/plugin/framework/migrating/providers/index.mdx

```diff
@@ -0,0 +1,379 @@
+    return map[string]provider.ResourceType{
+        "tls_private_key":         &privateKeyResourceType{},
+        /* ... */
+    }, nil
+}
+
+func (p *provider) GetDataSources(_ context.Context) (map[string]provider.DataSourceType, diag.Diagnostics) {
+    return map[string]provider.DataSourceType{
+        "tls_public_key":  &publicKeyDataSourceType{},
+        /* ... */
+    }, nil
+}
+
+func (p *provider) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
+    return tfsdk.Schema{
+        Attributes: map[string]tfsdk.Attribute{
+            "proxy": {
+            /* ... */
+            },
+        },
+    }, nil
+}
+
+func (p *provider) Configure(ctx context.Context, req provider.ConfigureRequest, res *provider.ConfigureResponse) {
+    /* ... */
+}
+```
+
+## Provider Schema
+
+A provider schema defines the attributes and behaviors of the provider itself. For example, a provider that connects to
+a third-party API may define attributes for the base URL or a required authentication token.
+
+### SDKv2
+
+In SDKv2, you implement a provider Schema by populating the `Schema` field on the `schema.Provider` struct. The `Schema`
+field contains a `map[string]*schema.Schema`. Each map entry represents the name of the attribute and pointer to a
+`schema.Schema` struct that defines that attribute's behavior.
+
+The following example defines the provider schema in the `Schema` field within the `schema.Provider` struct.
+
+```go
+func New() *schema.Provider {
+    return &schema.Provider{
+        Schema: map[string]*schema.Schema{},
+        /* ... */
+```
+
+### Framework
+
+In the Framework, the `GetSchema` function returns the provider schema. The `GetSchema` function is part of the
+`provider.Provider` interface that your provider must implement. `GetSchema` returns a struct containing fields for
+`Attributes` and `Blocks`. These `Attributes` and `Blocks` contain `map[string]tfsdk.Attribute` and
+`map[string]tfsdk.Block`, respectively. Refer to [Providers - GetSchema](/plugin/framework/providers#getschema) in the
+Framework documentation for details.
+
+The following code shows the `GetSchema` function, which returns the provider schema.
+
+```go
+func (p *provider) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diagnostics) {
+    return tfsdk.Schema{}, nil
+}
+```
+
+Refer to the [Attributes](/plugin/framework/migrating/attributes-blocks#attributes) and
+[Blocks](/plugin/framework/migrating/attributes-blocks#blocks) pages in this migration guide to learn how to migrate
+those fields to the Framework.
+
+### Migration Notes
+
+Remember the following differences between SDKv2 and the Framework when completing the migration.
+
+In SDKv2, `schema.Schema` is a struct that defines attributes and behaviors (e.g., `Type`, `Optional`). In the
+Framework `tfsdk.Schema` is a struct that defines `Attributes`, which are structs that define attributes and behaviors
+(e.g., `Type`, `Optional`).
```

Comment:
```suggestion
- In SDKv2, `schema.Schema` is a struct that defines attributes and behaviors (e.g., `Type`, `Optional`). In the
Framework `tfsdk.Schema` is a struct that includes `tfsdk.Attributes`, which are structs that define attributes and behaviors
(e.g., `Type`, `Optional`).
```

Three small changes:
- Kinda weird to have a list with one bullet, but this matches the pattern in the rest of the guide.
- I think "includes" is more correct than "defines"?
- If we're going to use the full name for the other structs, do so for `tfsdk.Attributes` too.